### ISSUE TITLE
Unable to fetch parkrun event status data

### ISSUE THREAD

**Issue by pwhettlock** [fraz3alpha/running-challenges#366]
Since mid November, the Running Challenges plugin has been displaying the following:
 
```
! Unable to fetch parkrun event status data: Stats and Challenges, e.g. Regionnaire, may include events that haven't started yet !

Refresh the page to try again
```

Have located this message in the source code, but not been able to trace it back to the root cause. So, not aware if it's a data glitch, or a format change on the parkrun site.

Haven't noticed anything amiss with my profile, but assume it may impact others. @fraz3alpha / @lauracowen any thoughts?

**Comment by fraz3alpha** [comment]
@pwhettlock , I assume this was transitory and worked later? It does happen now and again, as it fetches it from the wiki page, and that is often missing for a bit (I assume while it gets redeployed)

**Comment by fraz3alpha** [comment]
Oh! Which URL do you use to go to your profile?

**Comment by pwhettlock** [comment]
https://www.parkrun.org.uk/parkrunner/88720/all/

**Comment by fraz3alpha** [comment]
Fixed in v1.4.0, uploaded yesterday. Seems to be available in Chrome and Firefox now
